src/js/sdk.tsx

```diff
@@ -65,15 +65,18 @@ export function init(passedOptions: ReactNativeOptions): void {
     // eslint-disable-next-line deprecation/deprecation
     ?? passedOptions.transportOptions?.bufferSize
     ?? DEFAULT_OPTIONS.maxQueueSize;
+
+  const enableNative = passedOptions.enableNative !== undefined
+    ? passedOptions.enableNative
+    : NATIVE.isNativeAvailable();
```

Comment:
This looks good, now that it's not dependent on the transport, but the `const enableNative` should be false also in the case when `passedOptions.enableNative === true` and `NATIVE.isNativeAvailable() === false`